core/src/main/scala/org/apache/pekko/persistence/jdbc/query/scaladsl/JdbcReadJournal.scala

```diff
@@ -119,7 +117,8 @@ class JdbcReadJournal(config: Config, configPath: String)(implicit val system: E
   override def persistenceIds(): Source[String, NotUsed] =
     Source
       .repeat(0)
-      .flatMapConcat(_ => delaySource.flatMapConcat(_ => currentPersistenceIds()))
+      .throttle(1, readJournalConfig.refreshInterval)
+      .flatMapConcat(_ => currentPersistenceIds())
```

Comment:
can we simply use currentPersistenceIds() as source and throttle on it?

Reply:
the default behavior is fail the stream, need change the throttle mode.

Reply:
It can't, i guess... because this is an infinite stream. if we use something like:

```scala
  override def persistenceIds(): Source[String, NotUsed] =
    currentPersistenceIds()
      .throttle(1, readJournalConfig.refreshInterval)
      ....
```

It only queries once, and then throttles the set of ids in flight.

Reply:
right, should have expanded the code further.